orthopairs/src/main/java/org/reactome/release/orthopairs/OrthopairFileGenerator.java

```diff
@@ -0,0 +1,93 @@
+package org.reactome.release.orthopairs;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.json.simple.JSONObject;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.*;
+
+public class OrthopairFileGenerator {
+    private static final Logger logger = LogManager.getLogger();
+    // Create source-target protein mapping file
+    public static void createProteinHomologyFile(String speciesKey, String sourceTargetProteinMappingFilename, JSONObject speciesJSON, Map<String,Set<String>> speciesProteinHomologs ) throws IOException {
+
+        logger.info("\tGenerating " + sourceTargetProteinMappingFilename);
+        File sourceTargetProteinMappingFile = new File(sourceTargetProteinMappingFilename);
+        if (sourceTargetProteinMappingFile.exists()) {
+            sourceTargetProteinMappingFile.delete();
+        }
+        sourceTargetProteinMappingFile.createNewFile();
```

Comment:
When I comment out the above 4 lines, it only appends to existing files. Same if I only leave the `.createNewFile()` method

Reply:
See: https://docs.oracle.com/javase/8/docs/api/index.html?java/nio/file/Files.html 
The `write()` method takes an `OpenOption` argument, which could let you specify if you want to overwrite or append.

Reply:
Hmm, that's strange, though.  According to the JavaDocs, it should create or overwrite by default:

https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#write-java.nio.file.Path-byte:A-java.nio.file.OpenOption...-

Reply:
I suspect it's because this code is using `java.io.File` https://docs.oracle.com/javase/8/docs/api/index.html?java/io/File.html

NOT `java.nio.files`.